src/inference/src/dev/preprocessing/preprocessing.cpp

```diff
@@ -33,9 +33,11 @@ bool ov::pass::AddPreprocessing::run_on_model(const std::shared_ptr<ov::Model>&
         preproc.input(i).tensor().set_element_type(
             InferenceEngine::details::convertPrecision(input_info->getPrecision()));
 
-        std::stringstream stream;
-        stream << input_info->getLayout();
-        preproc.input(i).tensor().set_layout(ov::Layout{stream.str()});
+        if (input_info->getLayout() != InferenceEngine::Layout::BLOCKED) {
```

Comment:
Don't we need similar handling for `Layout::SCALAR` and `Layout::ANY`?

Reply:
@vladimir-paramuzov `ANY` can't arise there 

https://github.com/openvinotoolkit/openvino/blob/a4314faf292212d5b21544444b55c2eaac8af807/src/inference/src/ie_layouts.cpp#LL172C1-L189C2

Reply:
`SCALAR` was added